.gitignore

```diff
@@ -266,4 +266,8 @@ Session.vim
 
 # Private test configuration and binaries.
 config.ps1
-**/IISApplications
\ No newline at end of file
+**/IISApplications
+
+# WCF copied test project.json files
+src/System.Private.ServiceModel/tests/Scenarios/
+project.json
```

Comment:
warning - it seems like there's an extra line break here or something. I believe as written _all_ project.json are ignored. I think the correct thing might be 

```
src/System.Private.ServiceModel/tests/Scenarios/**/project.json
```